app/views/breakdowns/MatchBreakdown2019.js

```diff
@@ -1,44 +1,100 @@
 import React from 'react';
+import ReactNative from 'react-native';
 import {
-  View
+  View,
+  Text
 } from 'react-native';
 import BreakdownRow from '../../components/BreakdownRow';
+import images from '../../config/images';
+import ImageCount from '../../components/ImageCount';
 import MatchBreakdown from '../breakdowns/MatchBreakdown';
 import breakdownStyle from '../../styles/breakdown';
 
+// Override our Image and Text to have specific sizes
+const Image = ({ style, ...props }) => <ReactNative.Image style={[breakdownStyle.imageSize, style]} {...props} />;
```

Comment:
You don't need to do this here but - for the love of god let's DRY this

Reply:
We do, here:
https://github.com/the-blue-alliance/the-blue-alliance-react/pull/28/files#diff-c050433af16119a1bae81c576f1d18ddR20